lib/bankaccounts/checking-account.rb

```diff
@@ -0,0 +1,48 @@
+module BankAccounts
+
+  # CheckingAccount inherits from Account class
+  class CheckingAccount < Account
+    attr_accessor :checks_used
+
+    def initialize(id, initial_balance)
+      super
+      @min_balance = 0
+      check_initial_balance(initial_balance)
```

Comment:
`#check_initial_balance` is called on `#super` and again here. Can you think of a way of changing the order in which `#super` is called so it happens only once?